### ISSUE TITLE
Handling of Buttons changed

### ISSUE THREAD

**Issue by dhzl84** [marcokreeft87/room-card#109]
**Which version of Room Card?**
v1.07.00

**Describe the bug**
I used to use the `tap_action` `toggle` to activate a `button`.
Today I got an error message from HA that the service `button/turn_off` is not supported.

My Card Config:

```
type: custom:room-card
title: Draußen
show_state: false
info_entities:
  - sensor.temperatur_aussensensor
  - sensor.luftfeuchtigkeit_aussensensor
entities:
  - entity: button.garagentor
    tap_action:
      action: toggle

```

**To Reproduce**
Use a button with toggle

**Expected behavior**
Either toggle works for `button`, or a new action `press` is added specifically for buttons

** Containment Action **
I use a service call to press the button:
```
type: custom:room-card
title: Draußen
show_state: false
info_entities:
  - sensor.temperatur_aussensensor
  - sensor.luftfeuchtigkeit_aussensensor
entities:
  - entity: button.garagentor
    tap_action:
      action: call-service
      service: button.press
      service_data:
        entity_id: button.garagentor

```

**Comment by marcokreeft87** [comment]
When did it last work?

**Comment by dhzl84** [comment]
Hard to tell, I don't use it much as it seems. I didn't use it at least for tha past 3 weeks. I have no logger data beyond that.

**Comment by marcokreeft87** [comment]
Hmm ok because the actions are just passed to hass. I don't really do anything with them.

**Comment by dhzl84** [comment]
OK, so we need to use the service call in order to use buttons, right?
Maybe worth adding an example in the documentation to avoid further issues since buttons do not appear in the documentation yet.

**Comment by marcokreeft87** [comment]
You need to call the correct service that works for the button. Thats something you need to figure out yourself, not something that has anything to do with the card.

**Comment by dhzl84** [comment]
Sure and I already did, just thought having an example in the documentation could help others figuring it out faster.

**Comment by marcokreeft87** [comment]
Its already in there : https://github.com/marcokreeft87/room-card/wiki/Configuration#actions 
Check the link to the lovelace actions

**Comment by alexrmay91** [comment]
I installed today and ran into this same issue while configuring some room cards. I believe the toggle action is supported by button entities as I have other cards that use the same format mentioned above for them and they work perfectly fine:
```
tap_action:
    action: toggle
```
I also tested this by making a new button card for the button entity, it uses the same format and functions properly. yaml for button card config:
```
show_name: true
show_icon: true
type: button
tap_action:
  action: toggle
entity: button.nodered_821790affde5c453
```
Maybe this issue should be re-opened?
